packages/ember-intl/addon/services/intl.js

```diff
@@ -89,6 +89,15 @@ const IntlService = Service.extend(Evented, {
 
     return this.formatMessage(translation, ...args);
   },
+  
+  
+  tH(key, ...args) {
```

Comment:
Could be a symbol to avoid collision with arguments but complicates the API even more.

```
import { html } from 'ember-intl';
// ...
this.t('foo.bar', { [html]: true });
```

Perhaps we go with what you've done but rename it to `tHtml`.  Could you add tests to cover the feature?